java-plugin/pom.xml

```diff
@@ -16,21 +16,24 @@
     <url>https://github.com/green-code-initiative/ecoCode/tree/main/java-plugin</url>
 
     <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>ecocode-rules-specifications</artifactId>
+            <version>${project.version}</version>
+            <classifier>java</classifier>
+        </dependency>
 
         <dependency>
             <groupId>org.sonarsource.java</groupId>
             <artifactId>sonar-java-plugin</artifactId>
             <type>sonar-plugin</type>
+            <scope>provided</scope>
         </dependency>
 
         <dependency>
             <groupId>org.sonarsource.sonarqube</groupId>
             <artifactId>sonar-plugin-api</artifactId>
-        </dependency>
-
-        <dependency>
```

Comment:
why do you delete this dependency ? did you test java plugin without this dependency with real java test-project ?

Reply:
this dependency is not needed for `mvn verify`